README.md

```diff
@@ -261,65 +311,98 @@ If you are using a downstream implementation of GRUB2 (e.g. from Fedora or Debia
 
 Hint: run `objcopy --only-section .sbat -O binary YOUR_EFI_BINARY /dev/stdout` to get these entries. Paste them here. Preferably surround each listing with three backticks (\`\`\`), so they render well.
 *******************************************************************************
-[your text here]
+```
+sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md
+shim,4,UEFI shim,shim,1,https://github.com/rhboot/shim
+shim.flatcar,1,Flatcar Container Linux,shim,15.8-r1,security@flatcar-linux.org
+
+sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md
+grub,4,Free Software Foundation,grub,2.12,https://www.gnu.org/software/grub/
+grub.flatcar,1,Flatcar,grub2,2.12-flatcar3,https://github.com/flatcar/flatcar
+```
+We do not provide with fwupd, fwupdate, and systemd-boot.
 
 *******************************************************************************
 ### If shim is loading GRUB2 bootloader, which modules are built into your signed GRUB2 image?
 Skip this, if you're not using GRUB2.
 
 Hint: this is about those modules that are in the binary itself, not the `.mod` files in your filesystem.
 *******************************************************************************
-[your text here]
+The GRUB2 standard modules built into our image are:
```

Comment:
`search_part_label` is in both lists, standard and non-standard. Which one should it be in?

Reply:
Non-standard. I'll fix.